en/lessons/ecto/querying.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.0.3
+version: 1.0.4
```

Comment:
Let's call this version `1.1.0` since there's a decent amount of changes we'll want to get translated.